website/package.json

```diff
@@ -14,7 +14,7 @@
     "@hashicorp/react-button": "5.0.1",
     "@hashicorp/react-call-to-action": "3.0.1",
     "@hashicorp/react-content": "7.0.1",
-    "@hashicorp/react-docs-page": "13.1.1",
+    "@hashicorp/react-docs-page": "13.1.7-alpha.10",
```

Comment:
Will bump this to a stable release once these PRs are finalized and https://github.com/hashicorp/react-components/pull/220 is merged and published.

Reply:
✅ Bumped to stable